src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/TagParser.java

```diff
@@ -43,7 +43,7 @@
  * </p>
  *
  */
-class TagParser {
+/* default */ class TagParser {
```

Comment:
public 

all top level classes, should become public. Please apply this rule to all others.

Reply:
all done. Now only two occurrences of `default` are left:
`MainFrameModel.ParseMode#ParseMode` and `JavadocDetailNodeParser.ParseErrorMessage#ParseErrorMessage`